__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const request = require('supertest');
+const db = require('../db/connection.js');
+const data = require('../db/data/test-data/index');
+const seed = require('../db/seeds/seed.js');
+const app = require('../app.js');
+
+beforeEach(() => seed(data));
+
+afterAll(() => db.end());
+
+describe('1. GET /api/topics', () => {
+	test('status 200, responds with an array of park objects', () => {
+		return request(app)
+			.get('/api/topics')
+			.expect(200)
+			.then(({ body }) => {
+				const { topics } = body;
+				expect(topics).toBeInstanceOf(Array);
+				expect(topics).toHaveLength(3);
+				topics.forEach((topic) => {
+					expect(topic).toEqual(
+						expect.objectContaining({
+							description: expect.any(String),
+							slug: expect.any(String),
+						})
+					);
+				});
```

Comment:
That's a great test for happy path! Just need a test for the error msg when the path is invalid, I see you have the error all set up in your app to send back "page not found" which is great! Just needs a test to check it's working 👍